### ISSUE TITLE
Cannot install letsencrypt

### ISSUE THREAD

**Issue by Diet-mar** [freedombox/Plinth#486]
Cannot install letsencrypt in plinth 0.9.2. 
Error returned is "Fehler beim Installieren der Anwendung: packages not found None ".
Even after installing certbot, which provides letsencrypt (v) via apt.
Note letsencrypt was renamed to certbot. Maybe some correlation?

**Comment by Diet-mar** [comment]
I am not sure if this is related:

The symptoms:
Accessing to the ikiwiki is impossible (except by using the privacy mode of Iceweasel). 
Access to http is automatically redirected to https because of "Diese Website verwendet HTTP Strict Transport Security (HSTS), um mitzuteilen, dass Iceweasel nur über gesicherte Verbindungen mit ihr kommunizieren soll.". I guess this is a new setting in apache that only allows https? I fully support this approach, but there is a glitch:
The certificate however is just valid for localhost (because letsencrypt does not work) and it expired 1979. I think this is because the certificate was generated first thing after installing the FBx, i.e. before the correct time was set. So the system used "time zero" (i.e. 1970) as basis.
So in summary access to http is redirected to https and then an invalid certificate is found.

**Comment by jvalleroy** [comment]
We have HSTS enabled here:
https://sources.debian.net/src/freedombox-setup/0.9.1/data/etc/apache2/conf-available/freedombox.conf/
The problem is if you are using a Let's Encrypt certificate for a certain domain, then you switch to a self-signed certificate and access it through the same domain. If the browser saw the LE cert before, it will not accept the self-signed cert (or even let you add an exception).

You should be able to clear the setting for that domain though:
http://classically.me/blogs/how-clear-hsts-settings-major-browsers

**Comment by Diet-mar** [comment]
Thanks jvallery, I eventually managed to clear the browser settings.

Also, this bug can be closed as with the recent update 0.8.0-1 in stretch letsencrypt is working.
